package.json

```diff
@@ -17,7 +17,9 @@
     "lint": "eslint .",
     "lint:fix": "yarn lint --fix",
     "format": "prettier --write *.{js,md,json}",
-    "release-templates": "node ./scripts/release-templates"
+    "release-templates": "node ./scripts/release-templates",
+    "release": "standard-version && git push --follow-tags origin master && npm publish && conventional-github-releaser --preset angular",
+    "release:beta": "standard-version --prerelease beta && git push --follow-tags origin master && npm publish && conventional-github-releaser --preset angular"
```

Comment:
Is it intended to push the beta on the `latest` tag?

Reply:
Nope, I'm not sure how it should be handled for beta versions. What would you suggest?

Reply:
To publish the beta version on a different tag like `beta` or `next`. Otherwise every time we do a new beta it will be installed by default for the new users. Maybe not what we want with a beta.

Reply:
Thank you, will fix that.